tests/test_clean.py

```diff
@@ -348,3 +348,10 @@ def test_bold_node_in_link(self):
         self.assertEqual(
             clean_node(self.wxr, None, tree.children), "2ちゃんねる, italic"
         )
+
+    def test_protocol_relative_url(self):
+        # https://en.wikipedia.org/wiki/Wikipedia:Protocol-relative_URL
```

Comment:
I should have digged deeper about these. Because `//` is just deprecated (not forbidden), there will still be a lot of code that will use it -- and it is not necessarily true for every wikiproject that they have switched to hard `https://`.